src/transformers/models/auto/modeling_auto.py

```diff
@@ -538,6 +538,10 @@
     ]
 )
 
+# TODO: Do the LayoutLM classes belong here, or should they only be in VisualQuestionAnsweringPipeline?
+# Because the bounding boxes are optional inputs to the model, you technically _can_ execute these models
+# through the question answering pipeline (because its output shape matches), but I'm not sure if models
+# should belong in multiple mappings.
```

Comment:
This comment can be removed, the LayoutLM classes should only be in MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES.

Reply:
Because the performance of the model significantly downgrades in case you're not providing bounding boxes.

Reply:
Yes I agree. I'll delete them from here.